mobile/mysterium/openvpn_connection_setup.go

```diff
@@ -45,31 +46,49 @@ func (wrapper *sessionWrapper) Wait() error {
 }
 
 type statsUpdater interface {
-	Save(stats stats.SessionStats)
+	Save(stats dto.SessionStats)
+}
+
+// StatUpdater takes in the state channel and reports stats to it
+// TODO: SEPARATE PR TO FIX THIS MESS
+type StatUpdater struct {
```

Comment:
Don't export anything which are not needed, this whole structs/functions/interfaces/should be package internal

Reply:
unexported